x/twap/client/cli/query.go

```diff
@@ -85,6 +119,30 @@ Example:
 	return cmd
 }
 
+// getQuoteDenomFromLiquidity gets the quote liquidity denom from the pool. In addition, validates that base denom
+// exists in the pool. Fails if not.
+func getQuoteDenomFromLiquidity(ctx context.Context, clientCtx client.Context, poolId uint64, baseDenom string) (string, error) {
```

Comment:
note: this was extracted from pre-existing code. No additional changes were made